test.hack

```diff
@@ -0,0 +1,13 @@
+namespace Hack\GettingStarted\MyFirstProgram;
+namespace HH\Lib\Str;
+
+<<__EntryPoint>>
+function main(): void {
```

Comment:
This currently defines `HH\Lib\Str\main` which is unwanted

Reply:
good feedback! was just messing around and accidentally added when I wasn't paying attention -- removed.